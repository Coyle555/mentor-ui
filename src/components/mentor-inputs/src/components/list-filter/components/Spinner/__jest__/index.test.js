import React from 'react';
import { Spinner } from '../index';
import renderer from 'react-test-renderer';

test('Default render of spinner', () => {
	const tree = renderer.create(<Spinner />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Render of spinner with a custom class', () => {
	const tree = renderer.create(<Spinner className="foo" />).toJSON();

	expect(tree).toMatchSnapshot();
});
