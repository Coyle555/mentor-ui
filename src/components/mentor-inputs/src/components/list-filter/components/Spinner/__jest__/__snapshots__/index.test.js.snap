// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default render of spinner 1`] = `
<i
  className="far fa-spinner mui-spinner "
/>
`;

exports[`Render of spinner with a custom class 1`] = `
<i
  className="far fa-spinner mui-spinner foo"
/>
`;
